README.md

```diff
@@ -0,0 +1,372 @@
+||  ec2.ca-central-1.amazonaws.com   |  opt-in-not-required  |  ca-central-1   ||
+||  ec2.ap-southeast-1.amazonaws.com |  opt-in-not-required  |  ap-southeast-1 ||
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+
+```
+
+### Install moactl
+
+Download the [latest release of moactl](https://github.com/openshift/moactl/releases/latest) and add it to your path.
+
+Verify your installation by running the following command:
+
+```
+$ moactl
+Command line tool for MOA.
+
+Usage:
+  moactl [command]
+
+Available Commands:
+  completion  Generates bash completion scripts
+  create      Create a resource from stdin
+  delete      Delete a specific resource
+  describe    Show details of a specific resource
+  edit        Edit a specific resource
+  help        Help about any command
+  init        Applies templates to support Managed OpenShift on AWS clusters
+  list        List all resources of a specific type
+  login       Log in to your Red Hat account
+  logout      Log out
+  logs        Show logs of a specific resource
+  verify      Verify resources are configured correctly for cluster install
+  version     Prints the version of the tool
+
+Flags:
+      --debug     Enable debug mode.
+  -h, --help      help for moactl
+  -v, --v Level   log level for V logs
+
+Use "moactl [command] --help" for more information about a command.
+```
+
+Run the following command to verify that your AWS account has the necessary permissions:
+
+```
+$ moactl verify permissions
+I: Validating SCP policies...
+I: AWS SCP policies ok
+```
+
+If you encounter errors, check the [required AWS permissions](required-aws-permissions.md) documentation for more details.
+
+Verify that your AWS account has necessary quota to deploy an OpenShift cluster. Sometimes quota varies by region, which may prevent you from deploying:
+
+```
+$ export AWS_DEFAULT_REGION=us-west-2 && moactl verify quota
+I: Validating AWS quota...
+E: Insufficient AWS quotas
+E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid
+```
+
+If needed, try another region:
+```
+$ export AWS_DEFAULT_REGION=us-east-2 && moactl verify quota
+I: Validating AWS quota...
+I: AWS quota ok
+```
+
+If you need to increase your quota, navigate to your [AWS console](https://aws.amazon.com/console/), and request a quota increase  for Amazon Elastic Compute Cloud (Amazon EC2) > On-Demand Standard (A, C, D, H, I, M, R, T, Z) to 200.
```

Comment:
For now I would drop everything after "Amazon Elastic..." and say something like "and request a quota increase for the service that failed".